Modules/Microsoft365DSC/Examples/Resources/EXOPhishSimOverrideRule/1-Create.ps1

```diff
@@ -0,0 +1,38 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        EXOPhishSimOverrideRule "EXOPhishSimOverrideRule-_Exe:PhishSimOverr:d779965e-ab14-4dd8-b3f5-0876a99f988b"
+        {
+            Comment                                  = "Comment note";
+            Credential                               = $Credscredential;
```

Comment:
We should only pass in AppId, Tenant and certThumbprint here since this is what were are receiving and. Credentials is not defined here and should be removed.